### PR TITLE
Update the Openchain version

### PR DESCRIPTION
**Pull request by Flavien**
### Changelog
* Update the Openchain version from 0.5.0 to 0.6.2.

### Description of the change

Openchain 0.6 is based on the RTM version of .NET Core 1.0, whereas 0.5 was based on RC1. This change updates the version being deployed by the template to 0.6.2.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@Flavien__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: openchain-blockchain-coinprism/azuredeploy.parameters.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
             "value": "Standard_A0"
         },
         "openchainVersion": {
-            "value": "0.5.0"
+            "value": "0.6.2"
```
Comment (bmoore-msft) [inline openchain-blockchain-coinprism/azuredeploy.parameters.json]: Just remove this from the param file since it defaults to the only allowed version

Comment (Flavien) [inline openchain-blockchain-coinprism/azuredeploy.parameters.json]: I'd need to add it back next time when two versions will be supported. Is there any harm in having it here?

Comment (bmoore-msft) [inline openchain-blockchain-coinprism/azuredeploy.parameters.json]: no harm, just unnecessary when there is a default version... any interactive deployment experience will handle the extra versions when available.

Comment (Flavien) [inline openchain-blockchain-coinprism/azuredeploy.parameters.json]: Got it. I just removed it.

**Pull request merged**